### DISCUSSION TITLE
Fuzzy glob completion?

### DISCUSSION THREAD

**Discussion by spikespaz** [marlonrichert/zsh-autocomplete#421] [Q&A] [answered]
I have read the required documentation as well as a portion of the source but do not feel confident to begin the trial-and-error process of figuring this out myself.

When I configured ZSH manually before using this plugin with the help of `newuserinstall` and `compinit`, I was able to do the following:

`d/g/s/d<tab>` completes to `~/Documents/github.com/spikespaz/dotfiles`

What is the correct way to *add* this behavior atop the existing options configured by this plugin without side-effects?

I do not know which completion feature enables this.

Thank you.

**Comment by marlonrichert** [comment, marked as answer]
```
zstyle ':completion:*:paths' path-completion yes
```

See https://zsh.sourceforge.io/Doc/Release/Completion-System.html#index-path_002dcompletion_002c-completion-style

**Reply by spikespaz** [reply]
Thanks!

**Reply by marlonrichert** [reply]
No problem. Be aware, though, that it can potentially freeze if you have typos in the path. Pressing `^C` should get you out of it, though.
